short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py

```diff
@@ -176,6 +176,10 @@ def compute_slice(slice_idx):
                 }
             )
         )
+        # if all intermediate json files are empty, close collated json file
+        json_sizes = sum([os.path.getsize(tfn) for tfn in output_json_filenames[:-1]])
```

Comment:
The `sum([` is redundant you can drop the `[`

Reply:
Fixed thanks!